### DISCUSSION TITLE
Change password & Token expiration date

### DISCUSSION THREAD

**Discussion by tudordanbalas** [onepub-dev/onepub#537] [Q&A] [answered]
Hello,

I have couple of questions: 
- For accounts that used email and password to create their own account, is there a way to reset or change the password?
- Does the Onepub CI/CD access token has an expiration date? The one you get by pressing 'Export token'

Thanks!

**Comment by bsutton** [comment, marked as answer]
inline

On Thu, Jul 4, 2024 at 6:29 PM Tudor-Dan Balas ***@***.***>
wrote:

> Hello,
>
> I have couple of questions:
>
>    - For accounts that used email and password to create their own
>    account, is there a way to reset or change the password?
>
> Yes. there is a standard forgotten password prompt on the login screen.

[image: image.png]



>
>    - Does the Onepub CI/CD access token has an expiration date? The one
>    you get by pressing 'Export token'
>
> > All OnePub tokens, whether obtained during a login or via the onepub
export command, expire after 30 days.
full details: https://docs.onepub.dev/workflows/setting-up-ci-cd#expiry
